client/src/main/java/com/vesoft/nebula/client/storage/StorageClientImpl.java

```diff
@@ -56,69 +58,53 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(StorageClientImpl.class);
 
-    private Map<HostAndPort, StorageService.Client> clients;
+    private Map<HostAndPort, StorageService.Client> clients = new ConcurrentHashMap<>();
 
-    private MetaClient client;
-    protected List<TTransport> transports;
-    private Map<String, Map<Integer, HostAndPort>> leaders;
+    private MetaClientImpl metaClient;
```

Comment:
If possible please use interface.

Reply:
Maybe later, because I use get xxx from cache in `MetaClientImpl`